### ISSUE TITLE
Versions not tagged

### ISSUE THREAD

**Issue by fniephaus** [graalvm/mx#149]
It'd be great if new versions could be git-tagged. This would make it easier to check out specific versions of mx without having to browse through the commit history.

**Comment by zakkak** [comment]
@dougxc some changes break backwards compatibility and require changes in `suite.py`. Being able to seamlessly get a specific mx version that is known to work with a project would be nice. Maybe tagging major releases (that break backwards compatibility) would be enough.

**Comment by dougxc** [comment]
This is actually a reasonable request - I'm just balking because we don't currently have a good way for automatically tagging mx commits based on mx version ;-) We'll look into how to do this.

In the meantime, here's a shell command that gives you a rudimentary mapping from git commits to the mx version update they made:
```
git log -G'version = VersionSpec' --pretty='%H' | while read h; do echo $h $(git show $h | grep '+version = VersionSpec'); done
```

**Comment by dougxc** [comment]
Fixed by 0b1924f795e86410407e6211e005d38dc4b63ceb
